src/InstallationPage.css

```diff
@@ -92,7 +101,7 @@ body {
   font-size: 1.5rem;
   font-weight: 500;
   line-height: 1.4;
-  color: var(--sky-blue); /* ✨ Cyan text pop */
+  color: var(--secondary); /* ✨ Cyan text pop */
```

Comment:
This is a good change. You don't have to for this PR. But, removing this comment will help as well.